### DISCUSSION TITLE
social login user verification

### DISCUSSION THREAD

**Discussion by ricardoorfao** [fastapi-users/fastapi-users#895] [Q&A] [answered]
I would like to bypass the user verification if logging in using Google/Facebook. Well, maybe not bypass it per se, but verify all users that use social login. We don't need to verify users if Google or Facebook already did the job for us, right? How could be the best way to achieve this?

**Comment by frankie567** [comment, marked as answer]
The most straightforward way to do this would be to overload the `oauth_callback` method on the `UserManager`:

```py
class UserManager(BaseUserManager[UserCreate, UserDB]):
    # ...
    async def oauth_callback(
        self, oauth_account, request: Optional[Request] = None
    ):
        user = await super().oauth_callback(oauth_account, request)
        user.is_verified = True
        await self.user_db.update(user)
        return user
```

**Reply by ricardoorfao** [reply]
This is great. Thank you, @frankie567

**Reply by raphaeldemedion** [reply]
awesome
